### ISSUE TITLE
Setting `aria-current="false"` on a `<NavLink />` erroneously raises a prop warning

### ISSUE THREAD

**Issue by anied** [ReactTraining/react-router#7794]
## Version

5.2.0

## Test Case

https://codesandbox.io/s/react-router-forked-j3kvm

## Steps to reproduce

1. Create a `<NavLink />` element, and pass it the prop `aria-current="false"`

## Expected Behavior
Per the [react-router documentation](https://reactrouter.com/web/api/NavLink/aria-current-string) and the [source for `<NavLink />`](https://github.com/ReactTraining/react-router/blob/master/packages/react-router-dom/modules/NavLink.js), this should be an acceptable value for the `aria-current` prop.

## Actual Behavior
A prop warning is raised:
> Warning: Failed prop type: Invalid prop `aria-current` of value `false` supplied to `NavLink`, expected one of ["page","step","location","date","time","true"].

**Comment by anied** [comment]
For what it is worth, the reason we are doing this is because we are using `<NavLink />` elements as tabs in a tabpanel interface with deep linking, and in this case `aria-selected` is a more accurate state to use than `aria-current`.

Thanks very much for any assistance you can provide.

**Comment by timdorr** [comment]
It's fixed on master, but not yet released: #7503
